docs/docs/deploy/baremetal.md

```diff
@@ -242,6 +251,22 @@ yarn rw deploy baremetal --first-run
 
 If there are any issues the deploy should stop and you'll see the error message printed to the console. Assume it worked, hooray! You're deployed to BAREMETAL.
 
+### Starting on reboot
+The `pm2` service requires some system "hooks" to be installed so it can boot up using your systems service manager.  Otherwise, your services will need to be manually started again on reboot.  These steps only need to be run the first time you deploy to a machine.
+
+1. Ssh into your server as you did for the "Server Setup".  Navigate to your source folder.  For example `cd /var/www/example`
```

Comment:
I tend to make "SSH" either all caps, or all lowercase (like the command itself). Looks like I used a mix of both in this doc, but we should really settle on one or the other. Probably all caps like the man page does: https://linux.die.net/man/1/ssh